deploy/template.yaml

```diff
@@ -29,322 +26,295 @@ Parameters:
     Type: String
     Default: "none"
     Description: Secrets name prefix
-  AuditEventNamePrefix:
-    Description: "The audit event name prefix"
-    Type: AWS::SSM::Parameter::Value<String>
-    Default: "/common-cri-parameters/AuditEventNamePrefix"
-  CriIdentifier:
-    Description: "The unique credential issuer identifier"
-    Type: AWS::SSM::Parameter::Value<String>
-    Default: "/common-cri-parameters/CriIdentifier"
+  VpcStackName:
+    Type: String
+    Default: "platform-vpc"
```

Comment:
we have conflicting defaults here and in `samconfig.toml`

Reply:
I thought we were overriding what's here within samconfig for DEV deploys but I'll doublecheck :)